templates/javascript/spec/provider.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+describe('Service: <%= cameledName %>', function () {
+
+  // load the service's module
+  beforeEach(module('<%= scriptAppName %>'));
+
+  // instantiate service
+  var <%= cameledName %>,
+    init;
+
+  beforeEach(function () {
+    init = function () {
```

Comment:
why is this in the before each? nothing seems to be changing it so it could go in the main describe on line 10